compiler/src/dotty/tools/dotc/quoted/Interpreter.scala

```diff
@@ -213,8 +213,8 @@ class Interpreter(pos: SrcPos, classLoader0: ClassLoader)(using Context):
   private def loadClass(name: String): Class[?] =
     try classLoader.loadClass(name)
     catch
-      case MissingClassDefinedInCurrentRun(sym) =>
-        suspendOnMissing(sym, pos)
+      case MissingClassValidInCurrentRun((sym, origin)) =>
```

Comment:
Are those double parentheses needed?
```suggestion
      case MissingClassValidInCurrentRun(sym, origin) =>
```
Same twice more below.